docs/configurations.md

```diff
@@ -0,0 +1,216 @@
+
+| Name                                              | Description                                                                                      | Default                 |
+|---------------------------------------------------|--------------------------------------------------------------------------------------------------|-------------------------|
+| `scalar.db.server.port`                           | Port number for ScalarDB Server.                                                                 | `60051`                 |
+| `scalar.db.server.prometheus_exporter_port`       | Prometheus exporter port. Prometheus exporter will not be started if a negative number is given. | `8080`                  |
+| `scalar.db.server.grpc.max_inbound_message_size`  | The maximum message size allowed to be received.                                                 | The gRPC default value. |
+| `scalar.db.server.grpc.max_inbound_metadata_size` | The maximum size of metadata allowed to be received.                                             | The gRPC default value. |
+
+For details about ScalarDB Server, see [ScalarDB Server](scalardb-server.md).
+
+## Other ScalarDB configurations
+
+The following are additional configurations available for ScalarDB:
+
+| Name                                                             | Description                                                                                                                                                                                                       | Default              |
+|------------------------------------------------------------------|-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|----------------------|
+| `scalar.db.metadata.cache_expiration_time_secs`                  | ScalarDB has a metadata cache to reduce the number of requests to the database. This setting specifies the expiration time of the cache in seconds.                                                               | `-1` (no expiration) |
+| `scalar.db.active_transaction_management.expiration_time_millis` | ScalarDB maintains ongoing transactions, which can be resumed by using a transaction ID. This setting specifies the expiration time of this transaction management feature in milliseconds.                       | `-1` (no expiration) |
+
+### Two-phase commit support
+
+ScalarDB supports transactions with a two-phase commit interface. With transactions with a two-phase commit interface, you can execute a transaction that spans multiple processes or applications, like in a microservice architecture.
+
+For details about using two-phase commit, see [Transactions with a Two-Phase Commit Interface](two-phase-commit-transactions.md).
+
+## Configuration example - App and database
```

Comment:
Why did you remove the ScalarDB Server example here? In 3.8 and earlier, we don't deprecate it yet?

Reply:
Sorry, I forgot to re-add the example after realizing that we would address 3.8 first (the example was removed in the [previous PR that I closed](https://github.com/scalar-labs/scalardb/pull/1082/files/b3a9d9f0990323abef4a561d6db50c8beb931b82#r1322606138)). I've re-added the example.